lib/lknvball_web/schema/schema.ex

```diff
@@ -42,27 +42,42 @@ defmodule LknvballWeb.Schema do
       %Lknvball.Events.Event{}, _ ->
         :event
 
-      %Lknvball.Events.EventUsers{}, _ ->
+      %Lknvball.Events.EventUser{}, _ ->
         :event_user
 
       _, _ ->
         nil
     end)
   end
 
+  def context(ctx) do
+    loader =
+      Dataloader.new()
+      |> Dataloader.add_source(Lknvball.Accounts.User, Lknvball.Accounts.dataload_user())
```

Comment:
this is ok. I'm wondering if we should handle failures for dataload_user though just in case.

Reply:
Hmm. That seems like a reasonable idea.  I didn't really see anything in the docs about handling errors.  Kinda seems like it would just return something or null.